cfme/tests/cloud/test_instance_power_control.py

```diff
@@ -744,3 +745,56 @@ def test_terminate(self, provider, testing_instance,
             testing_instance.STATE_UNKNOWN
         )
         soft_assert(self.verify_vm_power_state(vm, terminated_states), "instance not terminated")
+
+
+@pytest.mark.meta(automates=[1701188, 1655477, 1686015, 1738584])
+def test_power_options_on_archived_instance_all_page(testing_instance):
+    """This test case is to check Power option drop-down button is disabled on archived and orphaned
+       instances all page. Also it performs the power operations on instance and checked expected
+       flash messages.
+       Note: Cloud instances can not be orphaned
+
+    Bugzilla:
+        1701188
+        1655477
+        1686015
+        1738584
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/2h
+        caseimportance: low
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.9
+        casecomponent: Control
+        tags: power
+        testSteps:
+            1. Add provider cloud provider
```

Comment:
please fix this

Reply:
Please open 1-liner PR for this change. PRT results are good, and I'd like to merge the newly automated tests.